.azure-pipelines/client.yml

```diff
@@ -0,0 +1,62 @@
+trigger:
+- master
+
+jobs:
+- job: 'Build_Target_Packages'
+
+  pool:
+    vmImage: 'ubuntu-16.04'  
+
+  steps:
+  - task: UsePythonVersion@0
+    displayName: 'Use Python 3.6'
+    inputs:
+      versionSpec: 3.6
+
+  - script: |
+     python -m venv .\
```

Comment:
On ubuntu `python` is always Python 2, and you need `python3`. I hope AzureDevOps didn't change that when you change spec to 3.6 that would be a terrible idea.
I suggest you explicitly use `python3`

Reply:
We have a call directly above it `Use Python 3.6`. I can call python3 explicitly, but this is the same thing.